templates/api.html

```diff
@@ -0,0 +1,167 @@
+						</span>
+					</p>
+					<ul style="list-style: none;">
+						<li>
+							Hover over rows and click on the +/- symbols to change your favorites(yellow)
+						</li>
+					</ul>
+				</div>
+				<h3>
+				API
+				</h3>
+				<div class="api">
+					<p>
+						You can query cdnjs via our API as below
+						<br><br>
+						Without any query parameters it will return the name and main file URL of every library on cdnjs:<br><br>
+						<code>https://api.cdnjs.com/libraries</code>
+					<br><br>
+					To search, use:
+					<br><br>
+				<code>https://api.cdnjs.com/libraries?search=[query]</code>
+				<br><br>
+				You can also "select" a certain library if you already know its name on CDNJS, e.g.
+				<br><br>
+			<code>https://api.cdnjs.com/libraries/[name]</code>
+			<br><br>
+			Without any query parameters it will return the whole meta data of the library, if you want to specify the info you want (recommended), use "fields" query:
+			<br><br>
+		<code>https://api.cdnjs.com/libraries/jquery?fields=name,filename,version</code><br><br>
+		All the usable fields is the same as the library's package.json on
+		<a href="https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/jquery/package.json">
+			CDNJS
+		</a>(except auto-update config, we'll only return the auto-update type (currently <code>npm or git</code>)
+		<br><br>
+		API will reture minified result by default, if you wanna have a human readable result, try
+		<code>output=human</code>
+		like so:
+		<br><br>
+		<code>https://api.cdnjs.com/libraries?output=human</code><br><br>
+		<code>https://api.cdnjs.com/libraries?search=[query]?output=human</code>
+		<br><br>
+		If you would like more data, use the fields parameter which takes comma-separated values:
+		<br><br>
+		<code>https://api.cdnjs.com/libraries?search=[query]&fields=version,description</code>
+	<br><br>
+	To get a list of all files for that library, use the assets field:
+	<br><br>
+<code>https://api.cdnjs.com/libraries?search=[query]&fields=assets</code>
+<br><br>
+Other fields available are:
+<br>
+<pre><code>version
+description
+homepage
+keywords
+license
+repository
+autoupdate
+author
+assets
+</code></pre>
+<br>
+The API is served over Cloudflare with a hour expiry for requests.
+</p><br>
+<h1>Examples</h1>
+All Libraries (<a>https://api.cdnjs.com/libraries</a>) (Cut down a bit)<br>
+<pre><code class="json">{"results":[{"name":"1140","latest":"https://cdnjs.cloudflare.com/ajax/libs/1140/2.0/1140.min.css"},{"name":"1000hz-bootstrap-validator","latest":"https://cdnjs.cloudflare.com/ajax/libs/1000hz-bootstrap-validator/0.11.6/validator.min.js"},{"name":"10up-sanitize.css","latest":"https://cdnjs.cloudflare.com/ajax/libs/10up-sanitize.css/4.1.0/sanitize.min.css"},{"name":"16pixels","latest":"https://cdnjs.cloudflare.com/ajax/libs/16pixels/0.1.8/16pixels.min.css"},
+</code></pre>
+<br>
+All Libraries (Human Readable) (<a>https://api.cdnjs.com/libraries?output=human</a>) (Cut down a bit)<br>
```

Comment:
BTW, let's use a new line for the link, and remove `(`, `)`, so it'll look like:

All Libraries Human Readable output (Cut down a bit in the example)
https://api.cdnjs.com/libraries?output=human